contrib/src/templates/tera_templates.rs

```diff
@@ -8,7 +8,14 @@ use super::{TemplateInfo, TEMPLATE_DIR};
 lazy_static! {
     static ref TERA: Result<tera::Tera, String> = {
         let path: PathBuf = [&*TEMPLATE_DIR, "**", "*.tera"].iter().collect();
-        tera::Tera::new(path.to_str().unwrap()).map_err(|e| format!("{:?}", e))
+        let escape_ext = [".html.tera", ".htm.tera", ".xml.tera",
```

Comment:
Haha, 85 cols is the cutoff, eh?  Might be we're clashing because I've been trying for 80.   I didn't realize you were optimizing for minimizing line count.

I like putting this on two lines just to reinforce that the two lines of extensions are intended repetition:  If you added a different escaped extension you would expect to add an item to each line.  (Not that I imagine anyone would ever have a file named ".htm.tera".)